waspc/src/Wasp/AI/GenerateNewProject/PageComponentFile.hs

```diff
@@ -70,8 +120,7 @@ fixPageComponent newProjectDetails waspFilePath pageComponentPath queries action
               Fix these by replacing them with actual implementation.
             - If there are any duplicate imports, make sure to remove them.
             - There might be some invalid JS or JSX syntax -> fix it if there is any.
-            - All @wasp imports must come from the list below:
-              ${possibleImportsList}
+            - Remove redundant imports, but don't change any of the remaining ones.
```

Comment:
This line is probably redundant now.

Reply:
The line "- Remove redundant imports, but don't change any of the remaining ones."? I would leave it, to ensure it doesn't try to modify imports.